config/locales/idv/fr.yml

```diff
@@ -88,12 +88,9 @@ fr:
         warning_2: Vous devrez ressaisir vos informations personnelles, comme votre nom,
           votre numéro d’identification, etc.
     form:
-      activate_by_mail: Vérifiez plutôt votre adresse par courrier
       address1: Adresse
       address2: Adresse (optional)
-      city: Ville
-      no_alternate_phone_html: '<strong>%{link}</strong>. Nous vous ferons parvenir
-        une lettre contenant un code.'
+      city: Ville une lettre contenant un code.'
```

Comment:
@aduth looks like this got a little fudged -- I ran into it consolidating some i18n keys for an IPP page. Any concerns with me updating this like so? Is there any particular approval process I should go through?

```
city: Ville
```

Reply:
Whoops! Thanks for catching that slip-up @sheldon-b . I think it'd be fine to fix that directly, since the text had already existed and it was only by mistake that it had been combined with the removed text string.